runtime/moonbase/src/precompiles.rs

```diff
@@ -83,10 +83,46 @@ parameter_types! {
 	pub LocalAssetPrefix: &'static [u8] = LOCAL_ASSET_PRECOMPILE_ADDRESS_PREFIX;
 }
 
+#[precompile_utils::as_precompile_name]
```

Comment:
May be better as a derive macro + trait? (thinking about it, it could be the same for the precompile macro too?)

Reply:
A derive macro cannot work on a type alias. Here we need to create an enum from a tuple, it doesn't seem feasible with a derive macro.